src/Synapse/Synapse/Synapse.csproj

```diff
@@ -11,11 +11,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Analytics.Synapse.AccessControl" Version="1.0.0-preview.3" />
-    <PackageReference Include="Azure.Analytics.Synapse.Artifacts" Version="1.0.0-preview.10" />
     <PackageReference Include="Azure.Analytics.Synapse.ManagedPrivateEndpoints" Version="1.0.0-beta.3" />
-    <PackageReference Include="Azure.Analytics.Synapse.Spark" Version="1.0.0-beta.1" />
     <PackageReference Include="Microsoft.Azure.Management.Synapse" Version="2.1.0-preview" />
+    <PackageReference Include="Azure.Analytics.Synapse.Spark" Version="1.0.0-beta.1" />
+    <PackageReference Include="Azure.Analytics.Synapse.AccessControl" Version="1.0.0-preview.3" />
+    <PackageReference Include="Azure.Analytics.Synapse.Artifacts" Version="1.0.0-preview.9" />
```

Comment:
Please keep these lines to be exactly same with that of main branch.

Reply:
Done